### PR TITLE
chore(release): 1.2.0

### PR DESCRIPTION
**Pull request by adebayor123**
Release v1.2.0. Added a build target to execute the [standard-version](https://github.com/conventional-changelog/standard-version) utility. This bumps the version and updates the changelog.
---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,26 @@
...
+
+### Features
+
+* identify DOM events using CSS locator ([#87](https://github.com/aws-observability/aws-rum-web/issues/87)) ([1c911e0](https://github.com/aws-observability/aws-rum-web/commit/1c911e0bf14841567c8bffe8367fe4b5f4977c9f))
+
+
+### Bug Fixes
+
+* add Cache-Control max-age to CDN files ([#76](https://github.com/aws-observability/aws-rum-web/issues/76)) ([fad8fb9](https://github.com/aws-observability/aws-rum-web/commit/fad8fb9cc39fbd11b5e58501468b0a1971cc4279))
+* capture PerformanceNavigationTiming events even when window.load fires before plugin loads ([#81](https://github.com/aws-observability/aws-rum-web/issues/81)) ([ece1306](https://github.com/aws-observability/aws-rum-web/commit/ece1306d82f51453d4009701eed4051d91708810))
```
Comment (limhjgrace) [inline CHANGELOG.md:16]: It seems that we had a couple bug fixes also listed under the previous version as well. Are these duplicates intentional? 

#81 lines 16 & 33
#85 lines 17 & 34
#80 lines 18 and 35

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,26 @@
...
+* change RUM origin from AWS::RUM::Application to AWS::RUM::AppMonitor ([#85](https://github.com/aws-observability/aws-rum-web/issues/85)) ([ead3b41](https://github.com/aws-observability/aws-rum-web/commit/ead3b410bc7421c83bcd963d370ade7cbfb39a4e))
+* documentation typos ([#80](https://github.com/aws-observability/aws-rum-web/issues/80)) ([5492091](https://github.com/aws-observability/aws-rum-web/commit/549209115ac27d999820d5b6ff0c25586eb98ef3))
+* find the first script tag in head instead of the entire document ([#72](https://github.com/aws-observability/aws-rum-web/issues/72)) ([dc86ec6](https://github.com/aws-observability/aws-rum-web/commit/dc86ec609c5dadb975508faab0ce721147dca7e3))
+* Point to correct license files in bundle banner. ([#91](https://github.com/aws-observability/aws-rum-web/issues/91)) ([1082f23](https://github.com/aws-observability/aws-rum-web/commit/1082f2308ca7cb3b4bf99c00ce6bf60d56a52b1c))
```
Comment (limhjgrace) [inline CHANGELOG.md:20]: nit: can we unify capitalization style under this section?

File: CHANGELOG.md (hunk 1 above)
Comment (adebayor123) [reply CHANGELOG.md:16]: These are autogenerated by standard-version, but this seems to be a bug that has been reported years back. Will recreate a commit with the duplicates removed.

File: CHANGELOG.md (hunk 2 above)
Comment (adebayor123) [reply CHANGELOG.md:20]: Not under this section, but these are commit histories that we are incorporating in the version. The standard-version auto-generates this from commit history.

Comment (adebayor123) [reply CHANGELOG.md:20]: Synced offline, resolving this comment as these are from commit messages.

**Review by limhjgrace — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,23 @@
...
+* Set aws api call xray subsegment namespace to aws ([#90](https://github.com/aws-observability/aws-rum-web/issues/90)) ([36d4e3c](https://github.com/aws-observability/aws-rum-web/commit/36d4e3c822df113c310af4d1c03301c484182651))
+* Set x-amzn-trace-id header directly in request headers ([#93](https://github.com/aws-observability/aws-rum-web/issues/93)) ([706d93e](https://github.com/aws-observability/aws-rum-web/commit/706d93e84bd4ef1feb3e728d42c98be8cec10348))
+* Uncaught TypeError: Cannot read the properties of undefined (reading 'record') ([#75](https://github.com/aws-observability/aws-rum-web/issues/75)) ([0193480](https://github.com/aws-observability/aws-rum-web/commit/019348075d0c0aadf618f18eb89e09d827351bd5))
```
Comment (qhanam) [inline CHANGELOG.md:20]: This fix was part of 1.0.5.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,23 @@
...
+
+### Features
+
+* identify DOM events using CSS locator ([#87](https://github.com/aws-observability/aws-rum-web/issues/87)) ([1c911e0](https://github.com/aws-observability/aws-rum-web/commit/1c911e0bf14841567c8bffe8367fe4b5f4977c9f))
+
+
+### Bug Fixes
+
+* add Cache-Control max-age to CDN files ([#76](https://github.com/aws-observability/aws-rum-web/issues/76)) ([fad8fb9](https://github.com/aws-observability/aws-rum-web/commit/fad8fb9cc39fbd11b5e58501468b0a1971cc4279))
+* find the first script tag in head instead of the entire document ([#72](https://github.com/aws-observability/aws-rum-web/issues/72)) ([dc86ec6](https://github.com/aws-observability/aws-rum-web/commit/dc86ec609c5dadb975508faab0ce721147dca7e3))
```
Comment (qhanam) [inline CHANGELOG.md:16]: This fix was part of 1.0.5.

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,23 @@
...
+
+
+### Features
+
+* identify DOM events using CSS locator ([#87](https://github.com/aws-observability/aws-rum-web/issues/87)) ([1c911e0](https://github.com/aws-observability/aws-rum-web/commit/1c911e0bf14841567c8bffe8367fe4b5f4977c9f))
+
+
+### Bug Fixes
+
+* add Cache-Control max-age to CDN files ([#76](https://github.com/aws-observability/aws-rum-web/issues/76)) ([fad8fb9](https://github.com/aws-observability/aws-rum-web/commit/fad8fb9cc39fbd11b5e58501468b0a1971cc4279))
```
Comment (qhanam) [inline CHANGELOG.md:15]: This fix was part of 1.0.5.

File: CHANGELOG.md (hunk 2 above)
Comment (qhanam) [reply CHANGELOG.md:17]: This is a good note though. We should capitalize commit messages going forward.

**Review by qhanam — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -17,6 +31,9 @@ All notable changes to this project will be documented in this file. See [standa
 ## 1.0.5
 
 -   fix: Fixed uncaught TypeError when include/exclude URLs are used with tracing enabled
```
Comment (qhanam) [inline CHANGELOG.md:33]: nit: Remove this entry -- it is a duplicate of `fix: Uncaught TypeError...`

**Review by qhanam — APPROVED** [review]

**Pull request merged**